### DISCUSSION TITLE
Version 6.0

### DISCUSSION THREAD

**Discussion by Akascape** [Akascape/CTkDesigner-Support#22] [New Update]
# What's new in version 6.0?

![Screenshot](https://github.com/Akascape/CTkDesigner-Support/assets/89206401/2aacd119-c580-4840-baae-1b57a7a45845)

- New quick preview option
- Unlock option added
- Added navigation to segmented buttons
- Ability to change page of a widget
- Fixed spawn overlap issue with fixed widgets
- Added label in font scale
- Fixed reload image issue of label/button
- Added delay option in tooltip
- New document option in file menu
- Tried to reduce antivirus false detections, see [#20](https://github.com/Akascape/CTkDesigner-Support/discussions/20)
- More bug fixes

**Comment by Akascape** [comment]
@teijiIshida Test the new version, download available in the same page!

**Reply by teijiIshida** [reply]
Which page is it? When I go to Shop > CTkDesigner, it asks for payment again.

**Reply by Akascape** [reply]
@teijiIshida Open the order email you received and click the "go to your order" button. The download page will open again.

**Reply by teijiIshida** [reply]
Got it. Thanks for the fast update and reply.
